XamlControlsGallery/SettingsPage.xaml

```diff
@@ -56,9 +56,11 @@
                     <TextBlock Text="See the Sound page for how to enable sound in your app!" Style="{ThemeResource CaptionTextBlockStyle}" FontStyle="Italic" Margin="0,8,0,0"/>
                 </StackPanel>
                 <TextBlock Style="{StaticResource TitleTextBlockStyle}" Margin="0,40,0,0" FontWeight="Normal" Text="Navigation Style" />
-                <StackPanel Margin="0,10,0,0">
-                    <ToggleSwitch x:Name="navigationToggle" Header="Navigation control location" Toggled="navigationToggle_Toggled" OffContent="Left" OnContent="Top" Margin="0,0,0,4" />
-                </StackPanel>
+                <muxc:RadioButtons x:Name="navigationLocation" Margin="0,10,0,0"
+                        Header="Navigation location" SelectionChanged="navigationLocation_SelectionChanged">

```

Comment:
Hmm. I'm not sure if "Navigation location" is quite right. What about "Navigation position", or "NavigationView Display Mode"?

Reply:
I went with "Navigation position" as "NavigationView Display Mode" is a bit technical, though to be fair, this is an app for developers.